### DISCUSSION TITLE
Student Pack Application Error: School Affiliation Issue

### DISCUSSION THREAD

**Discussion by ouhoy** [community/community#114781] [GitHub Education] [answered]
### Hi everyone,

I'm trying to get the GitHub Student Pack but I'm running into an error. Here's what I have done so far:

- **Enrollment:** I'm a student at SIST Morocco British Education (Rabat, Morocco), associated with Cardiff Metropolitan University (Cardiff, UK).
- **Documentation:** I have a transcript confirming my student status provided by my campus.
- **Email:** My school email is provided by Cardiff Met.

I am currently attending my classes in Morocco while I am enrolled in Cardiff Metropolitan University.

### The errors I am getting: 

- Selected school doesn't have a campus location in my country. Which in my case does but not sure why it's not listed.
- Transcript needs to indicate virtual learning if remote. I am attending classes in my campus not online.
- If no campus location, document must indicate virtual learning. :/

### Actual Error: 

Your application cannot be reviewed until you fix the following:

- The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

- You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

**Comment by samus-aran** [comment, marked as answer]
Hi @ouhoy  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional campuses.

Hope this helps! Let us know how you get along 👍

**Reply by ouhoy** [reply]
Hello there,

Thank you for your time and support!

I have submitted the ticket in the given form.

Kind regards

**Reply by 77x** [reply]
Hey there I have a similar issue, did you manage to solve yours promptly?

Cheers

**Reply by ouhoy** [reply]
Hello @77x ,

Yes, I have managed to solve the issue. I have created a ticket where I have explained everything in detail by providing my email and campus website and affiliation website.

You can submit your here too: https://support.github.com/contact/education

If you need any further help please let me know.

Kind regards
